src/main/scala/org/constellation/domain/transaction/TransactionValidator.scala

```diff
@@ -4,21 +4,30 @@ import cats.data.ValidatedNel
 import cats.effect.Sync
 import cats.syntax.all._
 import org.constellation.schema.transaction.{LastTransactionRef, Transaction}
+import scodec.bits.Bases.Alphabets.Base58
 
 object TransactionValidator {
   type ValidationResult[A] = ValidatedNel[TransactionValidationError, A]
 
+  val base58Chars = (('1' to '9') ++ ('A' to 'Z') ++ ('a' to 'z'))
+    .filterNot(c => List('O', 'I', 'l').contains(c))
+    .toSet
```

Comment:
Could we maybe reuse Base58 object implemented in keytool and so use the same "lib" used for encoding to also validate the base58 chars?